infra/src/main/kotlin/fi/fta/geoviite/infra/linking/Linking.kt

```diff
@@ -171,6 +171,7 @@ data class TrackLayoutKmPostSaveRequest(
     val gkLocationConfirmed: Boolean,
     val gkLocationSource: KmPostGkLocationSource?,
     val gkLocation: GeometryPoint?,
+    val sourceId: IntId<GeometryKmPost>?,
```

Comment:
Tällä käytännössä mahdollistetaan linkityksen purkaminen, asettamalla tämä nulliksi linkitys purkautuu. Olen vähän kahden vaiheilla tästä, toisaalta haluaisin että kilometripylvään linkityksen purkaminen ei vaadi erillistä linkityksen purkukutsua, mutta toisaalta sen tekeminen pelkällä propsulla on myös melko pelottavaa. Asiaa pohdittuani päädyin että tämä olisi parempi vaihtoehto ihan jo siksikin että haluaisin kilometripylvään muiden propsujen tallennuksen ja linkityksen purun olevan yhdessä atominen operaatio, mutta muitakin näkökantoja saa esittää

Reply:
Pikaisella pohdinnalla vaikuttaa ihan hyvältä näin